riot-basics/rtc/.solution/main.c

```diff
@@ -45,7 +45,11 @@ static void *blink_thread(void *arg)
         rtc_get_time(&time);
         printf("Current time: %d:%d:%d\n",
                time.tm_hour, time.tm_min, time.tm_sec);
-        time.tm_sec += 5;
+        time.tm_sec  += 5;
```

Comment:
This change should not be part of this PR, so just remove the corresponding commit. I'll push another (better) fix later.